fiftyone/utils/sam3.py

```
@@ -545,3 +551,800 @@ def _forward_pass_auto(self, imgs):
+        else:
+            self._curr_exemplar_prompts = {}
+            self._curr_visual_prompts = []
+
+        if self.operation_mode == "concept":
+            outputs = self._forward_pass_concept(video_reader, sample)
+        else:
+            outputs = self._forward_pass_visual(video_reader, sample)
+
+        return outputs
+
+    def _forward_pass_concept(self, video_reader, sample):
```

Ruff (0.15.15)

[warning] Missing return type annotation for private function `_forward_pass_concept`

(ANN202)